docs/strictdoc.sdoc

```diff
@@ -180,6 +180,22 @@ of the `snap package format <https://snapcraft.io/docs/snap-format>`_.
 
 [/SECTION]
 
+[SECTION]
+TITLE: Installing StrictDoc as a brew package
+
+[FREETEXT]
+This installation variant is available in Mac OS and UNIX operating systems with
```

Comment:
Could you expand the UNIX into which systems are specifically supported by Brew? 1-2-3 most used examples would be enough.